sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2127,5 +2127,36 @@ ALTER TABLE `form_bronchitis` MODIFY `diagnosis3_bronchitis_form` text;
 ALTER TABLE `form_bronchitis` MODIFY `diagnosis4_bronchitis_form` text;
 #EndIf
 
+#IfColumn patient_data care_team
+ALTER TABLE `patient_data` DROP COLUMN `care_team`;
+#EndIf
+
+#IfMissingColumn patient_data care_team_provider
```

Comment:
and would then place this block before the prior code block

Reply:
So It should be like 
1. create care_team_provider 
2. copy care_team data to care_team_provider
3. drop care_team

RIGHT?

Reply:
yep, that is good